test/fixtures/compiled_with_docs.ex

```diff
@@ -26,6 +26,9 @@ defmodule CompiledWithDocs do
   # TODO: remove explicit guard: true when ~> 1.8
   defguard is_zero(number) when number == 0
 
+  @doc "&mdash; & &ndash; : EM-DASH & EN-DASH"
+  def example_with_special_HTML_chars_in_doc, do: nil
```

Comment:
Instead of adding a new function, maybe we can change one of the existing ones. Such as `example_1`? For example: "Another example with &mdash; & &ndash;"?

Reply:
agreed about not adding new function.

I'd go as far as saying that the existing test that we're changing:

```diff
- more &gt; than
+ more > than
```

is enough to catch regressions.

Reply:
fixed.